bgpd/bgp_vty.c

```diff
@@ -6616,6 +6617,53 @@ DEFUN (no_neighbor_timers_connect,
 	return peer_timers_connect_unset_vty(vty, argv[idx_peer]->arg);
 }
 
+DEFPY (neighbor_timers_delayopen,
+       neighbor_timers_delayopen_cmd,
+       "neighbor <A.B.C.D|X:X::X:X|WORD>$neighbor timers delayopen (1-240)$interval",
```

Comment:
I'm curious why only 240 seconds here?

Reply:
Could that happen that if I set holdtime for instance to 300?

Reply:
Yes, because as suggested in the RFC, the interval of the running HoldTimer (not the configured HoldTime value) gets set to ~4 minutes (+5/-0 secs) by DelayOpenTimer related mechanisms.

Reply:
@ton31337 resolved?